contracts/libraries/FloatingPoint.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+import "@openzeppelin/contracts-v4/utils/Strings.sol";
+
+pragma solidity ^0.8.0;
+
+contract FloatingPoint {
```

Comment:
I was having issues linking this as a library for some reason. I tried inlining OZ dependency but that didn't help.

Reply:
you had a public function, looks like this was the reason